traffic_ops/traffic_ops_golang/routing/routes.go

```diff
@@ -130,6 +130,9 @@ func Routes(d ServerData) ([]Route, http.Handler, error) {
 		 * 4.x API
 		 */
 
+		// SSL Keys
+		{api.Version{Major: 4, Minor: 0}, http.MethodGet, `sslkey_expirations$`, deliveryservice.GetSSlKeyExpirationInformation, auth.PrivLevelAdmin, []string{"ACME:CREATE", "ACME:READ"}, Authenticated, nil, 41357729075},
```

Comment:
Why make a new endpoint instead of just adding expiration info to the keys already returned by other endpoints?

Reply:
we dont really have an endpoint that would fill the need here since this is for all certs and not specific to a DS or CDN.  we could add these new fields to those other endpoints as well but this would require a new one anyway.  plus its also good because otherwise we'd return a ton of info we dont need (like all certs and private keys) and it would be a bigger security concern instead of just generic expiration info. i have it as `admin` now but it could be opened up to a lower auth level if we wanted to since it doesnt include the private info

Reply:
it's tough because CDNs aren't tenanted. I'm tempted to say we should just have `/ssl_keys` with DS and CDN filters. But that's a lot of work too.

btw, this should probably allow a trailing `/`, since very nearly all other endpoints do in APIv4.